packages/react-core/src/components/Content/examples/Content.md

```diff
@@ -49,7 +49,7 @@ For example, rather than nesting the `<List>` and `<Title>` components within `<
 
 ```
 
-Text components such as Text, TextList, TextListItem can be placed within a TextContent to provide styling for html elements, and additional styling options applied to the children.
+ Html elements wrapped by `<Content>` are styled by the Content component.
```

Comment:
```suggestion
 HTML elements wrapped by `<Content>` are styled by the content component.
```

Maybe it would make more sense to me if I could load the website preview, but I'm not fully understanding the purpose of this line. Is this different than what we've already stated above?